app/marketing/views.py

```diff
@@ -615,6 +619,44 @@ def email_settings(request, key):
     return TemplateResponse(request, 'settings/email.html', context)
 
 
+def slack_settings(request):
+
+    # setup
+    profile, es, user, is_logged_in = settings_helper_get_auth(request)
+    if not es:
+        login_redirect = redirect('/login/github?next=' + request.get_full_path())
+        return login_redirect
+
+    msg = ''
+
+    if request.POST and request.POST.get('submit'):
+        token = request.POST.get('token', '')
+        repos = request.POST.get('repos').split(',')
+        channel = request.POST.get('channel', '')
+        es.slack_token = token
+        es.repos = repos
```

Comment:
I'm actually checking if notfication needs to be sent using users github name:
```subscribers = EmailSubscriber.objects.filter(profile__handle=username, repos__contains=[repo])```
that way user will be able to get notification only for his own repos.
but I don't know how this will behave if user wants to monitor repos that belong to organization,
and I don't know how to test such use case.
Any thoughts on how I can test this use case?